src/Core/Data/BrewClubTestingInitializer.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace mbsoft.BrewClub.Data
+{
+	public class BrewClubTestingInitializer : System.Data.Entity.DropCreateDatabaseIfModelChanges<BrewClubContext>
+	{
+
+		protected override void Seed(BrewClubContext context)
+		{
+			// this is pretty bad-ass - EF will recognize when the database needs to be changed, and this initializer is used to make it add/drop
```

Comment:
Not quite sure I fully grasp what this does.  Can you explain it to me at some point?